### PR TITLE
Modified viewmodels and objects for social media

### PR DESCRIPTION
**Pull request by Lugiasc**
- the organizer is added to the attendees' list so they receive a pop token
- added variable and functions in home and lao detail viewmodel to handle the launch of social media from their respective activity
- added function to send chirp in the social media viewmodel

### CODE REVIEW

**Review by GabrielFleischer — COMMENTED** [review]
As of now, everything seems good, maybe add a small user feedback when an error occurs.

File: fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/Wallet.java [hunk 1]
```diff
@@ -37,7 +38,7 @@
  * This class represent a wallet that will enable users to store their PoP tokens with reasonable,
  * realistic security and usability.
  */
-public class Wallet {
+public class Wallet implements Serializable {
```
Comment (GabrielFleischer) [inline fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/Wallet.java:41]: Why does the wallet need to be serializable ?

File: fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/socialmedia/SocialMediaViewModel.java [hunk 2]
```diff
@@ -198,4 +232,57 @@ public void subscribeToChannel(String laoId) {
...
+                              getApplication().getApplicationContext(),
+                              R.string.toast_error_sending_chirp,
+                              Toast.LENGTH_LONG)
+                          .show();
+                    }
+                  },
+                  throwable -> Log.d(TAG, "timed out waiting for result on chirp/add", throwable));
+      disposables.add(disposable);
+    } catch (GeneralSecurityException | IOException e) {
+      Log.d(TAG, PK_FAILURE_MESSAGE, e);
```
[the remark this answers is not in the document]
Comment (Lugiasc) [reply fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/socialmedia/SocialMediaViewModel.java:285]: nice idea

File: fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/Wallet.java (hunk 1 above)
Comment (Lugiasc) [reply fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/Wallet.java:41]: I might have added it by error, I'm gonna check whether it is needed or not

**Review by GabrielFleischer — APPROVED** [review]
Very nice !

**Review by arnauds5 — APPROVED** [review]

**Review by jbsv — APPROVED** [review]

**Pull request merged**